src/app/workflow/executions/executions-tab.component.ts

```diff
@@ -202,4 +202,9 @@ export class ExecutionsTabComponent extends EntryTab implements OnChanges {
       this.validationsTable = [];
     }
   }
+
+  ngOnDestroy() {
```

Comment:
I don't think you need this... It's already implemented in the base class EntryTab.